spec/rails/doc/smart/openapi.yaml

```diff
@@ -104,10 +117,13 @@ paths:
                   type: string
                 database_id:
                   type: integer
+                no_such_field_request:
+                  type: boolean
```

Comment:
Note) This is added but does not appear in the`expected.json` because it is removed because of the pattern `4) paths.*.*.requestBody.content.application/json.example.*`